src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -465,7 +466,7 @@ export default {
 	 * The render function to display the component
 	 *
 	 * @param {Function} createElement The function to create VNodes
-	 * @returns {VNodes} The created VNodes
+	 * @return {VNodes} The created VNodes
```

Comment:
> The type 'VNodes' is undefined

Reply:
The thing is, I don't know what to do against it :see_no_evil: And since it's a warning only, I decided to just leave it as it is. Maybe something for a follow-up PR? Same for the other places.

Reply:
Ok, same, resolving :)